addon/components/basic-dropdown-content.ts

```diff
@@ -125,18 +125,14 @@ export default class BasicDropdownContent extends Component<Args> {
   @action
   animateOut(dropdownElement: Element): void {
     if (!this.animationEnabled) return;
-    let parentElement = dropdownElement.parentElement;
-    if (parentElement === null) return;
-    if (this.args.renderInPlace) {
-      parentElement = parentElement.parentElement
-    }
+    this.animationClass = this.transitioningInClass;
+    let parentElement = this.destinationElement;
```

Comment:
Not sure if this is the right fix, but what I was seeing is that the parentElement was always null on `will-destroy` calls. However, since we want to animate at the destination location I think this is right? Not sure 🤷‍♂️